applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -71,61 +76,96 @@ static void stream_stopped_cb(struct bt_audio_stream *stream)
 	int ret;
 	struct event_t event;
 
+	broadcast_src_started = false;
+
 	event.event_source = EVT_SRC_LE_AUDIO;
 	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_NOT_STREAMING;
 
 	ret = ctrl_events_put(&event);
 	ERR_CHK(ret);
 
 	LOG_INF("Broadcast source stopped");
+
+	if (delete_broadcast_src && (broadcast_source != NULL)) {
+		ret = bt_audio_broadcast_source_delete(broadcast_source);
+		ERR_CHK_MSG(ret, "Unable to delete broadcast source");
+
+		broadcast_source = NULL;
+
+		LOG_INF("Broadcast source deleted");
+
+		delete_broadcast_src = false;
+	}
 }
 
 static struct bt_audio_stream_ops stream_ops = { .sent = stream_sent_cb,
 						 .started = stream_started_cb,
 						 .stopped = stream_stopped_cb };
 
+static void initialize(void)
+{
+	static bool initialized;
+
+	if (!initialized) {
+		for (size_t i = 0U; i < ARRAY_SIZE(streams); i++) {
+			streams[i].ops = &stream_ops;
+		}
+
+		initialized = true;
+	}
+}
+
 int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
 {
-	return 0;
+	LOG_DBG("Not possible to get config on broadcast source");
+	return -ENXIO;
 }
 
 int le_audio_volume_up(void)
 {
-	return 0;
+	LOG_DBG("Not possible to increase volume on/from broadcast source");
+	return -ENXIO;
 }
 
 int le_audio_volume_down(void)
 {
-	return 0;
+	LOG_DBG("Not possible to decrease volume on/from broadcast source");
+	return -ENXIO;
 }
 
 int le_audio_volume_mute(void)
 {
-	return 0;
+	LOG_DBG("Not possible to mute volume on/from broadcast source");
+	return -ENXIO;
 }
 
 int le_audio_play(void)
 {
+	broadcast_src_paused = false;
+
 	return 0;
 }
 
 int le_audio_pause(void)
 {
+	broadcast_src_paused = true;
```

Comment:
Yes, the data is just thrown away. I have removed this boolean.
Stopping the ISO but keeping the per adv is currently not possible, since syncable_cb() is called continously on the headsets, even if gateway is stopped. A fix must most likely be made in the lower layers before we can implement this in the application.

Reply:
Isn't that the way it should be? As long as we are listening to the per adv we should get syncable_cb. Then you could use this boolean (or some other way if possible) to allow the headset to call sink_sync only if !paused && !streaming

Reply:
AFAIK BIGinfo should not be sent when gateway is stopped, and then the syncable callback should not be called on headsets